examples/colour-uniform/main.rs

```diff
@@ -1228,15 +1230,18 @@ impl<B: Backend> ImageState<B> {
             cmd_buffer.begin();
 
             let image_barrier = m::Barrier::Image {
-                states: (i::Access::empty(), i::Layout::Undefined)
-                    .. (i::Access::TRANSFER_WRITE, i::Layout::TransferDstOptimal),
+                src_access: i::Access::empty(),
```

Comment:
we already have a nice `type State = (Access, Layout)` for images, let's use it here